src/shared/thorchain/client.ts

```diff
@@ -0,0 +1,19 @@
+import { Network } from '@xchainjs/xchain-client'
+import { ClientUrl, getDefaultClientUrl } from '@xchainjs/xchain-thorchain'
+
+import { envOrDefault } from '../utils/env'
+
+export const getClientUrl = (): ClientUrl => {
+  const { node: nodeTestnet, rpc: rpcTestnet } = getDefaultClientUrl()[Network.Testnet]
+  const { node: nodeMainnet, rpc: rpcMainnet } = getDefaultClientUrl()[Network.Mainnet]
+  return {
+    [Network.Testnet]: {
+      node: envOrDefault(process.env.REACT_APP_TESTNET_THORNODE_API, nodeTestnet),
+      rpc: envOrDefault(process.env.REACT_APP_TESTNET_THORNODE_RPC, rpcTestnet)
+    },
+    [Network.Mainnet]: {
+      node: envOrDefault(process.env.REACT_APP_MAINNET_THORNODE_API, nodeMainnet),
+      rpc: envOrDefault(process.env.REACT_APP_MAINNET_THORNODE_RPC, rpcMainnet)
+    }
+  }
+}
```

Comment:
All copied from `src/renderer/services/thorchain/common.ts` - #1739 will remove sources in `common` later